assets/dev/js/editor/elements/views/container.js

```diff
@@ -428,7 +439,7 @@ const ContainerView = BaseElementView.extend( {
 
 	onAddChild() {
 		this.$el.removeClass( 'e-empty' );
-		this.model.set( 'isInner', this.getNestingLevel() > 0 );
+		this.model.set( 'isInner', this.checkIfParentIsWidget() || this.getNestingLevel() > 0 );
```

Comment:
@ArmanPurtoian Is this call not double? I mean, when adding a child `onRender()` is called anyway, so can you then just remove this line?